flasksrc/schema.sql

```diff
@@ -85,3 +86,23 @@ CREATE TABLE toynet_survey_types (
   PRIMARY KEY (type_id)
   FOREIGN KEY (type_id) REFERENCES toynet_survey_questions (type_id)
 );
+
+CREATE TABLE toynet_topos (
+  topo_id INTEGER NOT NULL, --primary key
+  topology TEXT NOT NULL, --XML file (starting file for a lesson module)
+  author_id INTEGER NOT NULL,
+  PRIMARY KEY (topo_id)
+);
+
+CREATE TABLE toynet_sessions (
+  session_id INTEGER NOT NULL, --primary key
+  topo_id INTEGER NOT NULL, --links to the original topography for the lesson module (foreign key)
+  topology TEXT NOT NULL, --XML file (not the path)
```

Comment:
I find the comment a little imprecise, since it is the XML representation of the topology and not related to any file.